apps/sms/views/conversation/js/information.js

```diff
@@ -339,6 +346,51 @@ var VIEWS = {
           l10nContainsDateSetup(this.receivedTimestamp, message.timestamp);
           l10nContainsDateSetup(this.sentTimestamp, message.sentTimestamp);
           setL10nAttributes(this.sentTitle, 'message-sent');
+
+          var units = {
+            months: MONTH,
+            days: DAY,
+            hours: HOUR,
+            minutes: MINUTE,
+            seconds: SECOND
+          };
+          var lateness = message.timestamp - message.sentTimestamp;
+          var showLatenessNotice = message.timestamp && message.sentTimestamp &&
+            lateness >= LATENESS_DISPLAY_LOWER_LIMIT;
+
+          if (showLatenessNotice) {
+            [
+              'months',
+              'days',
+              'hours',
+              'minutes',
+              'seconds'
+            ].forEach((unit) => {
+              var output = 0;
+              var unitElement = this[Utils.camelCase('lateness-' + unit)];
```

Comment:
OK I understand now :) I'll do the change.

BTW Wouldn't it make sense to lazy query select, replacing this :

```javascript 
this.elements.forEach(function(name) {
  this[Utils.camelCase(name)] = this.panel.querySelector('.' + name);
}, this);

//this.contactList.setAttributes...
```
...by this :
```javascript 
this.elements.forEach(function(name) {
  var element;

  this[Utils.camelCase(name)] = () => {
    if (!element) {
      element = this.panel.querySelector('.' + name);
    }
    
    return element;
  };
}, this);

//this.contactList().setAttributes..
```